### PR TITLE
Blog app - controllers specs

### PR DESCRIPTION
**Pull request by 78fatemanazari**
Project requirements
Add RSpec as a gem to your project.
Create a Request spec file for all your controllers.
Make sure that for each action you check:
If response status was correct.
If a correct template was rendered.
If the response body includes correct placeholder text.

### CODE REVIEW

File: spec/requests/post_spec.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /index' do
+    before :each do
+      get '/users/1/posts'
+    end
+    it 'returns success status' do
+      expect(response).to have_http_status(:success)
+    end
```
Comment (1GORDON) [inline spec/requests/post_spec.rb:10]: Great job @78fatemanazari 🤝
- You are doing a great job on implementing the tests. However, this test case is still failing the check. Kindly fix it and ensure it passes the test check.
error

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @78fatemanazari ✋

Good job so far!

## Highlights!
- Has and uses RSpec gem for testings ✔️
- `index` and `show` actions for `User` and `Post` controllers are tested ✔️
- Spec files include tests that ensure the response body includes the correct placeholder text. ✔️
- Spec files include tests that ensure the correct view template is rendered ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/requests/post_spec.rb (hunk 1 above)
Comment (78fatemanazari) [reply spec/requests/post_spec.rb:10]: Thanks a lot, @1GORDON for your review, but all the tests are passed successfully inside my local machine.
Please check the screenshot below.
test

**Comment by 78fatemanazari** [comment]
Please, can you check my comments in the PR?

**Comment by 1GORDON** [comment]
Hello @78fatemanazari 🤝
- I have seen your comment. If you are very sure that your test cases are now passing, then you may resubmit your work for re-review. Thank you and happy coding!

**Review by OyePriscilla — APPROVED** [review]
Hi @78fatemanazari 👏 👏 👏 👏, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
All implemented test passed ✔️ 

Untitled

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**